docs/source/getting_started/overview.md

```diff
@@ -116,6 +116,18 @@ The [JupyterLab Language Server Protocol (LSP) integration](https://github.com/k
 
 By default, Elyra installs the [Python Language Server package](https://pypi.org/project/python-language-server/), enabling the productivity features in the notebook editor and the Python editor. Support for other languages can be added by manually installing additional Language Server packages and their prerequisites.
 
+##### R-Editor Language Server Protocol
```

Comment:
The R editor section in the new script editor topic might be a better location for this content because
 - it is too low level for an overview
 - it separates related content into multiple documents, forcing the user to look for related information in several locations (R LSP content is in the overview document but the R enterprise gateway content is in the editor document)

Reply:
moved to the script editor topic